src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/UWPApplication.cs

```diff
@@ -352,6 +352,7 @@ public void UpdatePath(Theme theme)
             }
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Globalization", "CA1307:Specify StringComparison", Justification = "something")]

```

Comment:
"something" is not a good justification ;)

Reply:
It’s a draft PR :)